File: Python_solutions/patterns/pattern5.py
#Pattern 5

#Given an integer n. You need to recreate the pattern given below for any value of N. Let's say for N = 5, the pattern should look like as below:

#*****
#****
#***
#**
#*
#Print the pattern in the function given to you.
#practice solution
n=5 
for i in range(n+1):
    for j in range(i, n):
        print("*" , end="")
    print()
#TUF SOLUTION :
  class Solution:
    def pattern5(self, n):
        for i in range(n+1):
            for j in range(i, n):
                print("*", end="")
            print()  

File: cpp_solution/patterns/pattern5.cpp
//Pattern 5

//Given an integer n. You need to recreate the pattern given below for any value of N. Let's say for N = 5, the pattern should look like as below:

//*****
//****
//***
//**
//*

//Print the pattern in the function given to you.
//practice solution: 
#include <iostream>
using namespace std;
int pattern5(int n){
    for(int i=1;i<=n;i++){
        for(int j =i;j<=n;j++){
            cout<<"*";
            }
        cout<<endl; 
        }
    }
int main(){
    int n =5;
    pattern5(n);
    return 0;
    }
//TIF SOLUTION :
class Solution {
public:
    void pattern5(int n) {
        for(int i=1;i<=n;i++){
            for(int j =i;j<=n;j++){
                cout<<"*";
            }
            cout<<endl; 
        }
    }
};